src/index.html

```diff
@@ -10,13 +10,103 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1> Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo_img"
+      >
+        <img
+          src="images/logo.png"
```

Comment:
The image source path uses backslashes (`\`) instead of forward slashes (`/`). Update the path to use forward slashes to ensure compatibility across all browsers.